maps/src/Map.tsx

```diff
@@ -43,14 +47,23 @@ function Markers({ filteredData, selfLocation }: { filteredData: MarkerData["map
 
   return (
     <>
+    {/* if comments are toggled, at high zoom levels clustering will be disabled (may be useful for taking ss/saving as pdf) */}
+    {/* For now it is disabled because spiderfy'ing lets us click really close map items. */}
+    <MarkerClusterGroup
+        // spiderfyOnMaxZoom={false}
+        maxClusterRadius={50}
+        animate={false}
+        // disableClusteringAtZoom={18}
+      >
```

Comment:
Some suggestions here:
- If there are less than X (maybe 30 or 50) locations, we can disable clustering completely.
- This is a bit of work, but it seems like we can create [custom cluster icons](https://yuzhva.github.io/react-leaflet-markercluster/?path=/docs/react-leaflet-markercluster--cluster-custom-icon#cluster-custom-icon). Current density color doesn't indicate anything for me really. Maybe we can use the color of the most common pin in the cluster.

Reply:
disableClusteringAtZoom does exactly that but using the zoom level instead of counting the items. I'll try it.
Idk about the most common pin color, I couldn't get why that one helps since the pins are scattered pretty much randomly.

Reply:
Increasing the value of disableClusteringAtZoom causes the frontend to crash, pretty much consistently.
Also maxClusterRadius was immutable, so at this point I gave up trying to implement your first suggestion, maybe you can take a look.